pkgs/servers/dns/bind/default.nix

```diff
@@ -33,6 +32,8 @@ stdenv.mkDerivation rec {
     "--without-python"
   ];
 
+  postInstall = "$out/sbin/rndc-confgen -r /dev/urandom -b 256 | head -n 5 > $out/etc/rndc.key";
```

Comment:
I didn't realize that. I've just removed the `postInstall` section from the commit.
I guess it should be added to the bind module instead, do you confirm ? If so, I'll submit another pull request for that.